src/VisualStudio/Core/Def/Implementation/Remote/JsonRpcClient.cs

```diff
@@ -58,8 +59,8 @@ public async Task InvokeAsync(string targetName, IReadOnlyList<object> arguments
 
             try
             {
-
-                await _rpc.InvokeWithCancellationAsync(targetName, arguments, cancellationToken).ConfigureAwait(false);
+                // convert to array until the bug is fixed - https://github.com/microsoft/vs-streamjsonrpc/issues/272
+                await _rpc.InvokeWithCancellationAsync(targetName, arguments?.ToArray(), cancellationToken).ConfigureAwait(false);
```

Comment:
You know the fix for this is likely to get inserted into VS today.

Reply:
[VS insertion PR](https://dev.azure.com/devdiv/DevDiv/_git/VS/pullrequest/181335?_a=overview).

Reply:
ah.. thank you. probably I can remove this workaround then.